src/Components/Home.js

```diff
@@ -34,6 +40,7 @@ function Home() {
           onClick={() => goToLinkHandle(user.username)}
         />
       ))}
+      {showProgress && <progress />}
```

Comment:
We are using PrimeReact, which Provides a progressbar out of the box. We should use this, to be consistent in the UI.
See the docs for the progressbar [here](https://primefaces.org/primereact/showcase/#/progressbar)

Reply:
Thanks, @Cahllagerfeld for the feedback. I have made the suggested changes.